iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/NodeBreakerVoltageLevel.java

```diff
@@ -590,7 +590,10 @@ public int getNode2(String switchId) {
 
         @Override
         public Terminal getTerminal(int node) {
-            return graph.getVertexObject(node);
+            if (graph.vertexExists(node)) {
+                return graph.getVertexObject(node);
+            }
+            return null;
```

Comment:
Is it OK to add a convenient `public Optional<Terminal> getOptionalTerminal(int node)`. Returning null could introduce a lot of breaking changes in the client code. I think the optional version is safer.